### PR TITLE
Added replace_spelling

### PR DESCRIPTION
**Pull request by naneja**

### CODE REVIEW

File: transformations/replace_spelling/transformation.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    def generate(self, sentence: str):
+        perturbed_texts = generate_sentences(text=sentence,
+                                            prob=0.20,
+                                            seed=self.seed,
+                                            max_outputs=self.max_outputs,
+                                            )
+        return perturbed_texts
+
+if __name__ == '__main__':
```
Comment (kaustubhdhole) [inline transformations/replace_spelling/transformation.py:59]: This can be commented/removed.

**Comment by kaustubhdhole** [comment]
Hi @naneja thanks for your changes. This looks good overall and satisfies the review criteria except for the "Data and Code Provenance" section missing in your README. 
Please pull main once in your branch.

**Comment by naneja** [comment]
Dear @kaustubhdhole Thank you very much for your review. I have updated transformation.py and readme. The readme now includes the data curation section and references section.

**Comment by naneja** [comment]
Dear @kaustubhdhole, can you please help as why test failed since it is getting passed at local machine. Seems error is "You should add a test file at this location /home/runner/work/NL-Augmenter/NL-Augmenter/transformations/transformation/test.json"

**Comment by naneja** [comment]
@kaustubhdhole @aadesh11 can you please review and help as why workflow getting failed while local test is getting passed.

**Pull request closed without merging**